packages/cyberstorm/src/components/componentStyles/MetaItem.module.css

```diff
@@ -16,13 +14,9 @@
 }
 
 /* Styles for MetaItem's text */
-.metaItemLabel {
-  font: normal 700 var(--text-xs) Inter, sans-serif;
-  line-height: var(--text-m);
-}
-
-/* Styles for MetaItem's icon */
-.metaItemIcon {
+.label {
+  font: var(--font-inter);
+  font-weight: var(--font-weight-bold);
   font-size: var(--text-xs);
```

Comment:
Setting font-size here instead of the root makes the icon static in size, unable to adapt to change.

Reply:
👍